Sources/Sentry/SentryProfiler.mm

```diff
@@ -306,126 +349,97 @@ + (void)startWithHub:(SentryHub *)hub
         return;
     }
 
-#    if SENTRY_HAS_UIKIT
-    [SentryDependencyContainer.sharedInstance.framesTracker resetProfilingTimestamps];
-#    endif // SENTRY_HAS_UIKIT
-
-    [_gCurrentProfiler start];
-
-    _gCurrentProfiler->_timeoutTimer = [SentryDependencyContainer.sharedInstance.timerFactory
-        scheduledTimerWithTimeInterval:kSentryProfilerTimeoutInterval
-                                target:self
-                              selector:@selector(timeoutAbort)
-                              userInfo:nil
-                               repeats:NO];
-#    if SENTRY_HAS_UIKIT
-    [[NSNotificationCenter defaultCenter] addObserver:self
-                                             selector:@selector(backgroundAbort)
-                                                 name:UIApplicationWillResignActiveNotification
-                                               object:nil];
-#    endif // SENTRY_HAS_UIKIT
```

Comment:
Lines 313-326 were moved from this class method into the (preexisting) instance method `-[SentryProfiler initWithHub:]` a bit above this in the diff..